src/pages/[movieId].js

```diff
@@ -0,0 +1,153 @@
+import styles from "@/styles/movie.module.scss";
+import { getMovie, getShowtimesForMovie } from "@/utils/dbFunctions";
+import dbConnect from "@/utils/dbConnect";
+
+export async function getServerSideProps(context) {
+  await dbConnect();
+
+  const { movieId } = context.query;
+  const movie = await getMovie(movieId);
+
+  const showtimes = await getShowtimesForMovie(movieId);
+
+  return {
+    props: {
+      showtimes: JSON.parse(JSON.stringify(showtimes)).sort(
+        (a, b) => new Date(a.date) - new Date(b.date)
+      ),
+      length: movie.length,
+      title: movie.title,
+      description: movie.description,
+      genre: movie.genre.join(", "),
+      director: movie.director.join(", "),
+      actors: movie.actors.join(", "),
+      country: movie.country,
+      year: new Date(movie.release).getFullYear(),
+      ageLimit: movie.age,
+      imageURL: movie.image
+    }
+  };
+}
+
+const Movie = ({
+  title,
+  description,
+  genre,
+  director,
+  actors,
+  country,
+  year,
+  ageLimit,
+  imageURL,
+  showtimes,
+  length
+}) => {
+  const showtimesList = showtimes.map((showtime) => {
+    const date = new Date(showtime.date);
+    return (
+      <li key={showtime}>
+        {date
+          .toLocaleDateString("sv-SE", {
+            weekday: "short",
+            day: "numeric",
+            month: "short"
+          })
+          .toUpperCase()}
+        <br />{" "}
+        {date.toLocaleTimeString("sv-SE", {
+          hour: "numeric",
+          minute: "numeric"
+        })}{" "}
+        <span>
+          {showtime.salong}
+          <br /> {length} min
+        </span>
+        <button className={styles["movie-tickets"]}>Biljetter</button>
```

Comment:
Even if the next page isn't done yet, maybe ad a Link to a placeholder page and send the relevant data with it for each showtime?